design/src/main/kotlin/uk/govuk/app/design/ui/component/Header.kt

```diff
@@ -42,31 +44,67 @@ fun TabHeader(
 
 @Composable
 fun ChildPageHeader(
-    text: String,
-    onBack: () -> Unit,
     modifier: Modifier = Modifier,
+    text: String? = null,
+    onBack: (() -> Unit)?,
+    onAction: (() -> Unit)?,
```

Comment:
We could default param to null here to save having to specify every time?

Reply:
Great suggestion - thanks @TimGJD 👍 - fixed up